### PR TITLE
Add support for test APIs

### PR DESCRIPTION
**Pull request by jamesnicholls**
🐿 v2.8.0

### CODE REVIEW

File: src/read/getUser.ts [hunk 1]
```diff
@@ -8,7 +8,8 @@ import { getSessionData } from './apis/session-service';
 import { validateOptions } from '../utils/validate';
 
 export const getUserBySession = async (
-	session: string
+	session: string,
+	testMode = false
```
Comment (doramatadora) [inline src/read/getUser.ts]: super minor but can this be typed as `:boolean` for annotations?

Comment (jamesnicholls) [reply src/read/getUser.ts]: Yeah but I didn't know how to do that with default values. Or will it default to false if I annotate it as a boolean?

**Comment by jamesnicholls** [comment]
@doramatadora Currently you can pass API host and key to the Auth API call, which can be used to call the Membership test environment instead of prod. I agree we should use the same mechanism for both the User API and Auth API. I would prefer to pass in the API host and key for both calls, but `n-memb-gql-client` (used to get the user profile) doesn't support that.

**Comment by doramatadora** [comment]
Discussed offline. Since we only use canned queries, we should move away from `n-memb-gql-client` and towards a simple `fetch` (GET). That will also make this package lighter and allow for consistency in specifying API environments.

**Pull request merged**